### ISSUE TITLE
Adding a option to create folders and subfolders when saving a imported deck

### ISSUE THREAD

**Issue by Arkewas** [nicho92/MtgDesktopCompanion#294]
Hello again,
as you can see from the attached screenshot when you wish to open a specific deck you find all decks in a single list. Adding the feature to create folders and subfolders when saving a deck (for example folders named with formats and / or colors) and find those folders when opening a deck will make easier to search for a specific deck.
An alternative could be to create filters by format and colors for the saved decks list, also including filters for MTG Arena- only formats  (Alchemy, Explorer, Historic and Timeless).
Probably the first option is more viable to implement in MTG Companion.

![MTG Companion saved decks](https://github.com/user-attachments/assets/2300a836-3150-430a-a11a-8185f6f6dfcd)

**Comment by nicho92** [comment]
Good idea. Currently you can only filtrer for MTG Arena compatible deck ( with the column filter to true)... 
Can you explain in few words, the Alchemy, Explorer, Historic and Timeles formats ? i'm not a arena player ;)

**Comment by Arkewas** [comment]
Essentially each format includes specific MTG sets and has a list of  Banned / Restricted cards.  In my opinion the most played formats are Historic and Explorer (they are also the only formats I play in MTG Arena) while Alchemy is the less played format.

For Timeless  https://mtg.fandom.com/wiki/Timeless

For Historic    https://mtg.fandom.com/wiki/Historic_(format)

For Explorer   https://mtg.fandom.com/wiki/Explorer

For Alchemy  https://mtg.fandom.com/wiki/Alchemy

**Comment by Arkewas** [comment]
If I have to make a point it's that it's not very clear how to install MTG Companion unless I missed some instructions. I didn't found them so I downloaded JDK Development Kit portable version from https://www.oracle.com/java/technologies/downloads/#jdk23-windows then I copied / merged the folders to MTG Companion's directory.

**Comment by Arkewas** [comment]
I've just joined MTG Companion Discord server (nickname Heris)

**Comment by nicho92** [comment]
Welcome :)

**Comment by Arkewas** [comment]
Thank you
